toqito/matrix_ops/tests/test_commutant.py

```diff
@@ -0,0 +1,38 @@
+"""Test the commutant function."""
+import numpy as np
+import pytest
+from toqito.matrix_ops.commutant import commutant
+
+@pytest.mark.parametrize(
```

Comment:
Can you also add test cases used from the "Examples" section here, please?
https://qetlab.com/Commutant